datamol/align.py

```diff
@@ -230,11 +223,15 @@ def auto_align_many(
             mol = mols[mol_id]
 
             if core_mol is not None:
+                # Only pass allowRGroups if set to True
+                # in order to support rdkit versions < 2021_03_1
+                # ref https://github.com/rdkit/rdkit/pull/3811
+                allowRGroups = {"allowRGroups": True} if allow_r_groups else {}
                 rdDepictor.GenerateDepictionMatching2DStructure(
                     mol,
                     reference=core_mol,
                     acceptFailure=True,
-                    allowRGroups=True,
+                    **allowRGroups,
                 )
```

Comment:
If RDKit changes its default to `allowRGroups=True` then we would call datamol with `allow_r_groups=False` but rdkit with `=True` silently.

I think it's best to instead check for the rdkit version and then decide to include `allowRGroups` as kwargs or not.